src/Sulu/Bundle/AdminBundle/Resources/js/components/FileUploadButton/FileUploadButton.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import Button from '../Button';
+import type {ButtonSkin} from '../Button';
+
+type Props = {|
+    children: string,
```

Comment:
why not allow any node or even null here (like the `Button` component does)?

Reply:
You are right, I am going to use the same type as in the `Button` component.